slack.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
```

Comment:
it looks like you made the branch off of master instead of staging and got some unrelated changes in your PR, can you redo the PR with a branch coming off from staging so we avoid merging errors when we push these changes from staging -> master